### DISCUSSION TITLE
app is missing for me?

### DISCUSSION THREAD

**Discussion by cfritze64** [johnlindquist/kit#36] [Q&A] [answered]
I can edit and run scripts from terminal, but I can't get an app interface to run. several of the simple commands (quit, update) give a No Parent Process error, I don't see a simple app icon in my system anywhere, no response to the shortcut defined in left-click.js 

Could this be Big Sur interfering with a permission?

**Comment by cfritze64** [comment, marked as answer]
just saw the new .dmg installer. Problem solved!

**Reply by johnlindquist** [reply]
Cool, there is a separation between "simplescripts" and the "simple" app which will be a challenge for me to message correctly.

I think 99% of people will use the app, so I may just hide the instructions on this site repo. 

(Originally, this was going to be a terminal-only project, but when I thought of all that I could offer with an App, my brain forced me to make it. Silly brain.)
